webcurator-docs/guides/system-administrator-guide.rst

```diff
@@ -241,241 +215,198 @@ data for the new QA module for each agency, and should be run* **once all
+-  the web curator webapp (webcurator-webapp.war)
+-  the web curator harvest agent (harvest-agent-h3.jar, harvest-agent-h1.jar)
+-  the web curator digital asset store (webcurator-store.war).
 
 Each of these three components must be deployed for the Web Curator
 Tool to be fully functional and more than one harvest agent can be
 deployed if necessary. Each Harvest Agent is capable of carrying out
 harvest actions. The more harvest agents deployed the more harvesting
 that can be done at any one point in time. The harvest agents and
 digital asset store can reside on any machine within the network, as
-they use SOAP over HTTP to communicate with each other.
-
-To deploy WCT to Tomcat:
+they use REST over HTTP to communicate with each other.
 
--  Make sure you have installed and configured both Java 1.8 JDK and
-   Apache-Tomcat 8.x.x successfully.
+To deploy WCT:
 
--  Set up the JMX Remote control and access files for the WCT core as described
-   in the section `JMX setup`_.
+-  Make sure you have installed and configured Java 1.8 JDK.
 
--  Deploy the WAR files into Tomcat. The simplest deployment is to
-   deploy all three WAR files into the same Tomcat container.
+-  Make sure you have installed and configured your database of choice, and
+   that it is now running.
 
-   -  You can copy the WAR files into the $TOMCAT_HOME/webapps/ directory.
-   -  Provided Tomcat is configured correctly, when you start Tomcat the
-      WAR files will be exploded and the application will start.
+-  Place the webapp, harvest-agent and store deployment files in the location
+   you wish to run them from. An additional logs directory will be created here
+   on startup.
 
--  Shut down Tomcat once the WAR files have been extracted. This will
-   allow you to modify the configuration files in the following steps.
+-  Start the WCT files using a standard Java command::
 
-Configure the Database Connection
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    java -jar webcurator-webapp.war
+    java -jar webcurator-store.war
+    java -jar harvest-agent-h3.war
 
-The open source version of the Web Curator Tool is configured to use a
-local PostgreSQL database. If you are using any other database, or are
-using a database server, you will need to change the database
-configuration.
+   These commands can be run in the foreground for testing, but it is
+   recommended to run them in the background, using a tool like Unix's
+   *nohup* command.
 
--  Set the correct database dialect in
-   TOMCAT/webapps/wct/WEB-INF/classes/\ **wct-core.properties**::
+-  To stop any WCT component, simply terminate the running process, or if running
+   in the foreground, simply use *Ctrl+c*.
 
-    #Hibernate Settings
-
-    hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
-    hibernate.default_schema=DB_WCT
+-  Before logging into WCT, *ensuring all components are shutdown*, modify the
+   configuration files in the following steps.
 
+Configure the Database Connection
```

Comment:
An easier way to configure the components would be to have users add a local application.properties file in the working directory of the java -jar command (e.g. application-local+mysql.properties for webapp, if we're using MySQL). This config file would only need to contain the properties from the built-in application.properties and application-local+mysql.properties that they want to override. Then the war and jar files wouldn't have to be altered (except if you want to change the default Heritrix profile).

In the case of webapp you can select the active profile with the command line parameter --spring.profiles.active=local+mysql. The other components don't have separate profiles, but just a single application-local.properties (that we can override in the same way as application-local+mysql.properties).